lib/rabl/template.rb

```diff
@@ -61,3 +61,23 @@ def self.call(template)
 
   ActionView::Template.register_template_handler :rabl, ActionView::Template::Handlers::Rabl
 end
+
+# Rails 6.X Template
+if defined?(ActionView) && defined?(Rails) && Rails.respond_to?(:version) && Rails.version.to_s =~ /^[6]/
+  module ActionView
+    module Template::Handlers
+      class Rabl
+        class_attribute :default_format
+        self.default_format = Mime[:json]
```

Comment:
@jpawlyn this needs to be in a `:json` format because of this change https://github.com/rails/rails/commit/0eb9e1e51a9df554e1cf7a42ec339ca6f0120a04

Reply:
I see there are some good tests for the different Rails versions but haven't been successful in running them. I'd really like to get some running against Rails 6.0.0.rc2 before making further changes. Will keep trying but any advice appreciated.